### PR TITLE
Update liclipse to 5.1.3

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by ran-dall** [comment]
@core-code Perhaps it's because it's hooking up to MediaFire to generate a key and do the download. When I run it through [VirusTotal (https://www.virustotal.com/#/url/b0e9c09973dc6f9b75a9f62c9ee1f08d1c85865b4a444f4e6fffa64e3c6e0ab8/detection) it picked up the MediaFire. When I looked up the sha256 sum of the file downloaded, it was the same as what you have on the original commit.

 ```bash
$openssl sha256 /Users/randall/Downloads/liclipse_5.1.3_macosx.cocoa.x86_64.dmg                                                      
SHA256(/Users/draque/Downloads/liclipse_5.1.3_macosx.cocoa.x86_64.dmg)= b1a89184a6185d4a9f393912ad1446b2d2bc017ea362bebe35fdefc2db50c8d7
```

**Comment by core-code** [comment]
thanks for the analysis! i'm still unsure how to work around this.

**Comment by ran-dall** [comment]
@core-code Hey man, @vitorgalvao [made the call](https://github.com/Homebrew/homebrew-cask/pull/55018#issuecomment-439085174) to remove casks from MediaFire. It's shame to see liclipse go..._maybe we could like them know_?

**Comment by core-code** [comment]
well i agree that mediafire is dubios at best and a very poor choice to host official software releases, especially when nowadays you can get "unlimited" bandwidth for 12$/year.

i'm not sure there is much we can do except file a bug report here:
https://www.brainwy.com/tracker/LiClipse/
and kindly ask for alternative downloads.

**Comment by ran-dall** [comment]
@core-code I guess it's the only option. Although it looks like we might be waiting for a mintue. lol.

**Comment by vitorgalvao** [comment]
https://github.com/Homebrew/homebrew-cask/pull/55071

**Comment by claui** [comment]
Hi @core-code! Your PR was doing fine with the modified, stable URL. (See also https://github.com/Homebrew/homebrew-cask/pull/55071#issuecomment-441126258)

Do you no longer want your PR merged?

**Comment by core-code** [comment]
sorry a "cask-repair -d" closed all my pull requests. don't know which other ones got lost

**Comment by core-code** [comment]
i'm afraid you need to walk me through that

File: Casks/liclipse.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 cask 'liclipse' do
-  version '4.0.0'
-  sha256 '4fc92d8b9165c3ac1b8df4d4f7a615e2950c949433f79c123734249892ce7762'
+  version '5.1.3,32gtr23s399ec2x'
+  sha256 'b1a89184a6185d4a9f393912ad1446b2d2bc017ea362bebe35fdefc2db50c8d7'
 
-  # mediafire.com/file/s5qczei26kot9z5 was verified as official when first introduced to the cask
-  url "https://www.mediafire.com/file/s5qczei26kot9z5/liclipse_#{version}_macosx.cocoa.x86_64.dmg"
+  # mediafire.com/file was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/liclipse.rb]: This might be problematic, as it isn’t specific enough. We’ll need to verify this one every time.

**Comment by vitorgalvao** [comment]
> i'm afraid you need to walk me through that

Don’t worry, I can squash on merge.

**Pull request merged**